### PR TITLE
Create 123.txt

### PR DESCRIPTION
**Pull request by vancuongngo**

### CODE REVIEW

File: 123.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+123
```
Comment (cuongngo-kry) [inline 123.txt:1]: ok

Comment (cuongngo-kry) [reply 123.txt:1]: one

**Review by cuongngo-kry — CHANGES_REQUESTED** [review]
you need to change

File: 123.txt (hunk 1 above)
Comment (vancuongngo) [reply 123.txt:1]: two

**Review by cuongngo-kry — COMMENTED** [review]
aha

**Review by cuongngo-kry — APPROVED** [review]
now itäs okey

**Pull request merged**